openpype/hosts/flame/otio/flame_export.py

```diff
@@ -0,0 +1,622 @@
+""" compatibility OpenTimelineIO 0.12.0 and newer
+"""
+
+import os
+import re
+import sys
+import json
+import logging
+import opentimelineio as otio
+from . import utils
+
+import flame
+from pprint import pformat
+
+reload(utils)  # noqa
+
+log = logging.getLogger(__name__)
+
+self = sys.modules[__name__]
```

Comment:
Try avoid using `sys.modules[__name__]` and if you need it then don't use `self` variable name. You're mixing "context" variables with constants.

Variable `marker_color_map` seems to be constant so should be named with upper case `MARKER_COLOR_MAP`.

The context specific variables is better to have at least class where are stored as class attributes.
```
class ContextData:
    fps = None
    seq_frame_start = None
    project = None
    clips = None
```
With this it is possible anywhere import
```
from ... import ContextData

ContextData.fps = 25
```
Also if some value change requires a validation or callback it can be easier modified to avoid direct access.
```
class ContextData:
    _fps = None
    seq_frame_start = None
    project = None
    clips = None
    
    @classmethod
    def _on_fps_change(cls):
        ...
    @classmethod
    def set_fps(cls, new_fps):
        if not isintance(new_fps, (int, float)):
            raise TypeError("Invalid fps type {}".format(type(new_fps)))
        if cls._fps != new_fps:
            cls._fps = new_fps
            cls._on_fps_change()
```

It is much easier for someone who does not know what is or should be used in which way to understand what can change and is not contstant but may change based on context.

Reply:
awesome, thanks for suggestion!